src/test/kotlin/org/bundleproject/ApplicationTest.kt

```diff
@@ -1,18 +1,63 @@
 package org.bundleproject
 
+import com.google.gson.Gson
 import io.ktor.http.*
 import io.ktor.server.testing.*
+import io.ktor.utils.io.charsets.*
+import kotlinx.coroutines.runBlocking
+import org.bundleproject.json.assets.ModAssets
+import org.bundleproject.json.responses.ModResponse
 import org.bundleproject.plugins.configureRouting
+import org.bundleproject.plugins.configureSerialization
+import org.bundleproject.utils.fetchAssets
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class ApplicationTest {
     @Test
-    fun testRoot() {
-        withTestApplication({ configureRouting() }) {
-            handleRequest(HttpMethod.Get, "/").apply {
+    fun testMod() {
+        withTestApplication({
+            configureRouting()
+            configureSerialization()
+        }) {
+            var assets: ModAssets
+            runBlocking {
+                assets = fetchAssets()
+            }
```

Comment:
Do tests not work with suspend? If they don't this can be changed to 
```kotlin
val assets = runBlocking { fetchAssets() }
```

Reply:
They do not, I tried

and ok, I forgot you can do that